apps/admin_app/assets/js/views/product/edit.js

```diff
@@ -35,6 +36,23 @@ export function imageOnEnter() {
   });
 }
 
+export function onToggleActivation() {
+  $(document).delegate('.variant_activate', 'change', function (event) {
```

Comment:
`delegate` has been deprecated, try to use alternative for this.
Follow the docs in this [link](http://api.jquery.com/delegate/)